src/frontend/css/style.scss

```
@@ -71,8 +71,9 @@ nav#primary-navigation {
   background-color: #eff3f6;
 }
 
-h2.heading {
-  font-size: 1.65rem;
+h1.heading {
+  @apply text-3xl;
+  @apply mb-2;
```

```suggestion
  @apply text-3xl mb-2;
```
I think it should be possible to write those assignments in one line.